File: samples/Ruby/arithmetic-integer.rb
puts 'Enter x and y'
x=gets.to_i  # to check errors, use x=Integer(gets)
y=gets.to_i

puts "Sum: #{x+y}",
     "Difference: #{x-y}",
     "Product: #{x*y}",
     "Quotient: #{x/y}",  # truncates towards negative infinity
     "Remainder: #{x%y}", # same sign as second operand
     "Exponentiation: #{x**y}"

File: samples/Go/arithmetic-integer.go
package main

import "fmt"

func main() {
    var a, b int
    fmt.Print("enter two integers: ")
    fmt.Scanln(&a, &b)
    fmt.Printf("%d + %d = %d\n", a, b, a+b)
    fmt.Printf("%d - %d = %d\n", a, b, a-b)
    fmt.Printf("%d * %d = %d\n", a, b, a*b)
    fmt.Printf("%d / %d = %d\n", a, b, a/b)  // truncates towards 0
    fmt.Printf("%d %% %d = %d\n", a, b, a%b) // same sign as first operand
    // no exponentiation operator
}
